docs/developer_guides/debugging_tools/benchmarking.md

```diff
@@ -71,6 +73,7 @@ The flags used in the benchmarking script are defined as follows:
 - `-m`: config name (e.g. `instant-ngp`). This should be the same as what was passed in for -c in the train script.
 - `-o`: base output directory for where all of the benchmarks are stored (e.g. `outputs/`). Corresponds to the `--output-dir` in the base `Config` for training.
 - `-t`: timestamp of benchmark; also the identifier (e.g. `2022-08-10_172517`).
+- `-s`: Launch a single job per GPU.
 - `-g`: specifies the gpus to use and if not specified (no -g flag), will automaticaly search for available gpus.
```

Comment:
Is this flag still used?

Reply:
That's a flag I added. It basically launches one job per GPU, then waits on the first one to be finished before relaunching on that GPU.
In the previous version, all jobs were launched in parallel at script launch (and Ctrl-C didn't kill the jobs). That works fine if you have several GPUs and lots of GPU memory, but not if you have a single 16Gb GPU.

Reply:
Ahh sorry, I was trying to highlight the `-g` flag.

Reply:
well this flag was simply ignored in the previous version of the scripts. It just took the list of remaining arguments as the list of GPUs, so I kept it that way and removed the unused flag. I'll adjust this doc.